Week1/README.md

```diff
@@ -1,42 +1,167 @@
-# Lesson 1: Retrieving Data
+# Lesson 1: MySQL and Node setup! Create, Insert and Select !
 
-In this class, students will be introduced to relational data terminology (row, column), the function of a primary key, and retrieving data from a MySQL database using SELECT queries.
+Objective : This module aims to incorporate JavaScript code to operate the MySQL database.
+MySQL client can be used to demonstrate SQL queries however, students should know how to
+make a MySQL database connection from JavaScript, run queries from JavaScript and
+capture results of queries in JavaScript.
 
-Objective: Students should be able to retrieve data from a database table using SELECT statements that include WHERE, GROUP BY, and ORDER BY.
-
-## Pre-Class Readings
+## Pre-Class Preparation
+- Install MySQL using the following [official docs](https://dev.mysql.com/downloads/mysql/)
+- MAC users may use `brew install mysql`
 
 Before arriving to class on Sunday, please watch all of the videos in [this video playlist](https://www.lynda.com/SharedPlaylist/0299ced540444d7197460e7f1f74ddab) on Lynda.
-- What are databases?
-- Exploring databases and database management systems
-- The features of a relational database
-- Introduction to database modeling
-- Using the basic SELECT statement
-- Creating SQL queries
-- Structuring the WHERE clause
-- Sorting query results
-- Using aggregate functions
-- Joining tables
+If you actually manage to watch them all and understand everything,
+then you are a database expert.
 
-## Pre-Class Preparation
-- Install MySQL windows guys can follow the [official docs](https://dev.mysql.com/doc/refman/5.7/en/windows-installation.html) unix users have it easyier as usual 😉
+## Setup
+
+### MySQL setup
+This setup assumes MySQL version 8.0.
+Windows users should use **Microsoft MySQL Command Line** client.
+Linux and MAC users should use **gnome-terminal** and **Terminal** respectively.
+**Microsoft MySQL Command Line** client gives you a **msql>** prompt after typing in your root password.
+Note that this password is the one you used for root user of the mysql.
+Linux and MAC users can execute `mysql -uroot -p` and then type the password.
+Following commands should be run under the **mysql>** prompt:
+```
+mysql> create user ‘hyfuser’@‘localhost’ identified with mysql_native_password by ‘hyfpassword’;
+# This command creates a user 'hyfuser' with password 'hyfpassword' for
+# the database server at 'localhost'
+
+mysql> grant all privileges on *.* to ‘hyfuser’@‘localhost’;
+# This command gives all permissions to user 'hyfuser'.
+(*.*) means every table of every database.
+
+mysql> create database userdb;
+This command creates a database 'userdb'
+```
+
+### Node setup
+This setup assumes that you have Node.js 0.6 or higher.
```

Comment:
This was the requirement of the mysqljs package. any version below 0.6 is not good for mysqljs. Most students would have higher versions because of previous javascript classes.
Perhaps I should mention that this is the node setup for mysql